src/Tasks/Common/build/Microsoft.NET.DefaultPackageConflictOverrides.targets

```diff
@@ -0,0 +1,233 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.DefaultPackageConflictOverrides.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(DisableDefaultPackageConflictOverrides)' != 'true'">
+    <PackageConflictOverrides Include="Microsoft.NETCore.App">
+      <OverridenPackages>
+        Microsoft.CSharp|4.4.0;
+        Microsoft.Win32.Primitives|4.3.0;
+        Microsoft.Win32.Registry|4.4.0;
+        runtime.debian.8-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.fedora.23-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.fedora.24-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.Apple|4.3.0;
+        runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.rhel.7-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.0;
+        System.AppContext|4.3.0;
+        System.Collections|4.3.0;
+        System.Collections.Concurrent|4.3.0;
+        System.Collections.Immutable|1.4.0;
+        System.Collections.NonGeneric|4.3.0;
+        System.Collections.Specialized|4.3.0;
+        System.ComponentModel|4.3.0;
+        System.ComponentModel.EventBasedAsync|4.3.0;
+        System.ComponentModel.Primitives|4.3.0;
+        System.ComponentModel.TypeConverter|4.3.0;
+        System.Console|4.3.0;
+        System.Data.Common|4.3.0;
+        System.Diagnostics.Contracts|4.3.0;
+        System.Diagnostics.Debug|4.3.0;
+        System.Diagnostics.FileVersionInfo|4.3.0;
+        System.Diagnostics.Process|4.3.0;
+        System.Diagnostics.StackTrace|4.3.0;
+        System.Diagnostics.TextWriterTraceListener|4.3.0;
+        System.Diagnostics.Tools|4.3.0;
```

Comment:
Wouldn't a better place for this list be in Microsoft.NETCore.App itself? In fact we already provide the assembly list we should just be able to provide extra metadata for the version.

Reply:
I think we should ship any updates to this that way, but we need this to work on already shipped versions of these packages.

Reply:
Yes, it is definitely a better place, but we have already shipped Microsoft.NETCore.App 2.0.0 and NETStandard.Library 2.0.0.  Thus we can't add this data to those packages.

This change allows data to come from those packages in the future, and that new data will either augment this (see [MergePackageOverrides](https://github.com/dotnet/sdk/pull/1805/files#diff-210154c344df4e4338c157d85a780a87R44)), or the future packages can turn the default data off all together by setting `DisableDefaultPackageConflictOverrides` and just provide its own data.